abci/server/grpc_server.go

```diff
@@ -43,7 +43,9 @@ func (s *GRPCServer) OnStart() error {
 	}
 
 	s.listener = ln
-	s.server = grpc.NewServer()
+	s.server = grpc.NewServer(
+		grpc.MaxRecvMsgSize(1024 * 1024 * 75),
```

Comment:
Out of curiosity, why was 75 MiB chosen for this value? 

[nit] for readability

```go
const mebibyte = 1024 * 1024
```

then here

```suggestion
		grpc.MaxRecvMsgSize(75 * mebibyte),
```